src/components/AppFrame/Help.test.js

```diff
@@ -36,14 +36,8 @@ describe("Help", () => {
 		modalRoot = document.createElement("div");
 		modalRoot.id = "modal";
 		document.body.appendChild(modalRoot);
-
-		windowOpen = window.open;
```

Comment:
Why the windowOpen?  In case it needs to be accessed?

Reply:
Lines were removed, not added.